docs/robots/jackal/tutorials_jackal.mdx

```diff
@@ -4,9 +4,1317 @@ sidebar_position: 3
+| `/feedback`          | `jackal_msgs/Feedback`       | High-frequency inputs from Jackal's encoders and motor current sensors.                                                                                                      |
+| `/status`            | `jackal_msgs/Status`         | Low-frequency status data for Jackal's systems. This information is republished in human readable form on the diagnostics topic and is best consumed with the Robot Monitor. |
+
+---
+
+## Jackal Software Setup {#jackal-software-setup}
+
+### Backing Up Robot Configuration
+
+Upgrading your Clearpath Jackal to ROS Noetic from older ROS distributions is a straightforward process;
+however it's important to understand that each Jackal is different, having undergone customization to your specifications.
+For more complete upgrade instructions see [this guide](https://clearpathrobotics.com/assets/guides/melodic/melodic-to-noetic/index.html).
+
+Please take the time to understand what these modifications are, and how to recreate them on your fresh install of Ubuntu Focal/ROS Noetic.
+
+#### Performing a Backup {#performing-a-backup}
+
+<ComponentPerformingABackup />
+
+### Installing and Configuring Robot Software
+
+:::note
+
+If you are upgrading your Jackal from an older version of ROS, please refer to our upgrade instructions [here](https://clearpathrobotics.com/assets/guides/kinetic/kinetic-to-melodic/index.html)
+and [here](https://clearpathrobotics.com/assets/guides/melodic/melodic-to-noetic/index.html).
+
+:::
+
+#### Installing Jackal Software
+
+:::note
+
+The physical Jackal robot comes pre-configured with ROS and the necessary Jackal packages already installed;
+therefore, you will only need to follow the instructions below if you are re-installing software on the Jackal.
+
+:::
+
+<ComponentInstallingRobotSoftware />
+
+#### Testing Base Configuration
+
+1. To test your configuration, start the background service with the following command:
+
+   ```
+   sudo systemctl start ros
+   ```
+
+2. The Comms indicator <img src="/img/robot_images/dingo_images/icon-comms.png" width="20" />
+   should turn green. You can check that the service has started correctly by checking the logs:
+
+   ```
+   sudo journalctl -u ros
+   ```
+
+3. Your Jackal should now be accepting commands from your joystick (see next section).
+   The service will automatically start each time you boot your Jackal's computer.
+
+#### Pairing the Controller {#pairing-the-controller}
+
+##### PS4 Controller
+
+<ComponentPs4ControllerPairing />
+
+### Setting up Jackal's Network Configuration {#jackal-networking}
+
+Jackal is normally equipped with a combination Wi-Fi + Bluetooth module.
+If this is your first unboxing, ensure that Jackal's wireless antennae are firmly screwed on to the chassis.
```

Comment:
Change to 
'...screwed onto the chassis.'
or
'...screwed to the chassis.'

Reply:
Done